static/js/containers/SourceListContainer.jsx

```diff
@@ -15,6 +15,9 @@ class SourceListContainer extends React.Component {
   }
 
   render() {
+    if (this.props.sourcesTableEmpty) {
+      return "The sources table is empty.";
```

Comment:
Here, we should render a standard messages when the source list has length zero.

The message about the source table being uninitialized should go into another component.  That component will render some text that points to the manual, how to ingest data, etc.